packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -634,6 +634,7 @@ class HotRunner extends ResidentRunner {
       if (!silent) {
         globals.printStatus('Restarted application in ${getElapsedAsMilliseconds(timer.elapsed)}.');
       }
+      unawaited(maybeCallDevToolsUriServiceExtension());
```

Comment:
It would be good to have a comment saying why it's ok to ignore errors from this future.